src/app/frontend/deploy/deploy.scss

```diff
@@ -18,3 +18,38 @@
   // TODO(bryk): Find a better, application wide constant for the min width.
   min-width: 600px;
 }
+
+.kd-deploy-label-list {
```

Comment:
I'm thinking about how to simplify this CSS and remove as much magic numbers as possible from here (e.g., 10px in a few places and this padding-bottom chain). I'll play with it tomorrow. But you may have your suggestions earlier.

The basic idea is to stick with stock Angular Material whenever possible.